backend/s3/s3.go

```diff
@@ -859,6 +875,9 @@ const (
 	defaultUploadCutoff = fs.SizeSuffix(200 * 1024 * 1024)
 	maxUploadCutoff     = fs.SizeSuffix(5 * 1024 * 1024 * 1024)
 	minSleep            = 10 * time.Millisecond // In case of error, start at 10ms sleep.
+
+	memoryPoolFlushTime = fs.Duration(time.Minute) // flush the cached buffers after this long
+	memoryPoolUseMmap   = false
```

Comment:
Why this is false by default?

Reply:
Default memory allocator is safer, only advances users knows what they are doing, so I;m leaving a way to set is via a configuration option.